osu.Game.Tournament/Screens/Gameplay/Components/TeamDisplay.cs

```diff
@@ -14,9 +14,21 @@ public class TeamDisplay : DrawableTournamentTeam
     {
         private readonly TeamScore score;
 
+        private bool showScore;
+
         public bool ShowScore
         {
-            set => score.FadeTo(value ? 1 : 0, 200);
+            get => showScore;
+            set
+            {
+                if (showScore == value)
+                    return;
+
+                showScore = value;
+
+                if (IsLoaded)
+                    score.FadeTo(value ? 1 : 0, 200);
```

Comment:
To avoid duplicate implementation of transforms, the usual pattern we use here is making a `private` update method and calling `FinishTransforms(true)` after the `LoadComplete` one. Not so important when there's only one, but might as well?

Reply:
I was trying to use `FinishTransforms()` here, but I must have botched it somehow. Should be more idiomatic now.